api-doc/swagger_v1.yaml

```
@@ -68,6 +70,8 @@ components:
         - $ref: "#/components/schemas/LiveVideo"
         - type: object
           properties:
+            published_at:
+              type: string
```

```suggestion
            published_at:
              type: string
              format: date-time
```
Add a format as specified here: [Data types](https://swagger.io/docs/specification/data-models/data-types/#:~:text=date-time)
Could be useful, though not really important. If it gets changed here, everywhere else should be changed too.